vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/HolderAgent.kt

```diff
@@ -302,20 +333,20 @@ class HolderAgent(
                     },
                 )?.let {
                     CandidateInputMatchContainer(
-                        inputMatching = it,
                         credential = credential,
+                        fieldQueryResults = it,
                     )
                 }
             }
         }
     }
 
-    private fun findDescriptorInputMatching(
+    private fun evaluateInputDescritorAgainstCredential(
```

Comment:
typo

Reply:
fixed in commit f85a5a54cacd343dd4bbdf10a5fbcd05dfb07d03